SConstruct

```diff
@@ -2575,15 +2575,29 @@ mantraWorldEnv =  houdiniEnv.Clone( IECORE_NAME="VRAY_ieWorld" )
 
 if doConfigure :
 
-	c = Configure( houdiniEnv )
+	# Since we only build shared libraries and not exectuables,
+	# we only need to check that shared libs will link correctly.
+	# This approach succeeds because building a shared library
+	# doesn't require resolving the unresolved symbols of the
+	# libraries that it links to.
+	houdiniCheckEnv = houdiniEnv.Clone()
+	houdiniCheckEnv.Append( CXXFLAGS = [ "-fPIC" ] )
+	houdiniCheckEnv.Append( LINKFLAGS = [ "-shared" ] )
+	c = Configure( houdiniCheckEnv )
 
-	if not c.CheckCXXHeader( "SOP/SOP_API.h" ) :
+	if not c.CheckLibWithHeader( "HoudiniGEO", "SOP/SOP_API.h", "CXX" ) :
 
 		sys.stderr.write( "WARNING : no houdini devkit found, not building IECoreHoudini - check HOUDINI_ROOT.\n" )
 		c.Finish()
 
 	else :
 
+		# Houdini 16.0 and beyond can optionally ship using Qt5.
+		# Since IECoreHoudini makes some UI related calls, we add
+		# a custom define so we can change the logic as needed.
+		if c.CheckLib( "Qt5Core", language="CXX" ) :
```

Comment:
This seems likely to accidentally find Qt5Core on the system, even if Houdini isn't using it. Perhaps we could just do a simpler `os.path.exists( "$HOUDINI_ROOT/lib/libQt5Core.so" )` or something along those lines?

Reply:
Ah, I guess I'm not hitting that because our system Qt is still Qt4...